pkg/lwip/contrib/_netif.c

```diff
@@ -83,7 +83,7 @@ int netif_get_opt(netif_t *iface, netopt_t opt, uint16_t context,
     return res;
 }
 
-int netif_set_opt(netif_t *iface, netopt_t opt, uint16_t context,
+int netif_set_opt(const netif_t *iface, netopt_t opt, uint16_t context,
```

Comment:
Are you sure, `const` is the right thing here for `set_opt`? The only reason the compiler does not throw any warning here, is because `iface` is cast to `lwip_netif_t` directly below. Maybe it works out, but having the target object be `const` on a set operation sounds like a future footgun to me.